api_tests/src/image.spec.ts

```diff
@@ -228,15 +236,19 @@ test("No image proxying if setting is disabled", async () => {
   ).toBeTruthy();
   expect(post.post_view.post.body).toBe("![](http://example.com/image2.png)");
 
-  let gammaPost = await resolvePost(delta, post.post_view.post);
-  expect(gammaPost.post).toBeDefined();
+  let betaPost = await waitForPost(
+    beta,
+    post.post_view.post,
+    res => res?.post.alt_text != null,
```

Comment:
```suggestion
    res => res?.post.alt_text !== null,
```

Reply:
This test fails with `!==` but passes with `!=`. Not sure why but Ive changed it back.